Multisig/App/SceneDelegate.swift

```diff
@@ -121,6 +121,14 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     // MARK: - Window Management
 
+    func presentForMain(_ controller: UIViewController) {
+        if windowState == .main {
+            mainWindow?.rootViewController?.present(controller, animated: true)
+        } else {
+            shouldBePresentedController = controller
```

Comment:
can we rename for a better name? This seems not specific -- what is the purpose of this view controller?

`delayedMainRootViewController` maybe

Reply:
renamed to delayedMainPresentedViewController